consensus/polybft/bridge_event_manager.go

```diff
@@ -103,19 +96,20 @@ type topic interface {
 	Subscribe(handler func(obj interface{}, from peer.ID)) error
 }
 
-// newStateSyncManager creates a new instance of state sync manager
-func newStateSyncManager(logger hclog.Logger, state *State, config *stateSyncConfig,
-	runtime Runtime) *stateSyncManager {
-	return &stateSyncManager{
-		logger:  logger,
-		state:   state,
-		config:  config,
-		runtime: runtime,
+// newBridgeEventManager creates a new instance of state sync manager
+func newBridgeEventManager(logger hclog.Logger, state *State, config *stateSyncConfig,
+	runtime Runtime) *bridgeEventManager {
+	return &bridgeEventManager{
+		logger:                 logger,
+		state:                  state,
+		config:                 config,
+		runtime:                runtime,
+		nextBridgeEventIdIndex: make(map[uint64]uint64),
 	}
 }
 
 // Init subscribes to bridge topics (getting votes) and start the event tracker routine
-func (s *stateSyncManager) Init() error {
+func (s *bridgeEventManager) Init() error {
```

Comment:
Change this in every function

```suggestion
func (b *bridgeEventManager) Init() error {
```